### ISSUE TITLE
Plugin doesn't load correctly in Rider 2020.1 EAP6

### ISSUE THREAD

**Issue by caveman-dick** [microsoft/azure-devops-intellij#321]
Currently the PR and WorkItems tabs show the following message:

![image](https://user-images.githubusercontent.com/516600/78240219-9df31b80-74d6-11ea-9ceb-f8939179d9ff.png)

with a stack trace in the fatal error log of:

```
VcsPullRequestContentProvider: unexpected fatal error initializing the controller

java.lang.NoSuchFieldError: Add
	at com.microsoft.alm.plugin.idea.common.ui.common.tabs.TabFormImpl.createActionsGroup(TabFormImpl.java:176)
	at com.microsoft.alm.plugin.idea.common.ui.common.tabs.TabFormImpl.ensureInitialized(TabFormImpl.java:112)
	at com.microsoft.alm.plugin.idea.git.ui.pullrequest.VcsPullRequestsForm.<init>(VcsPullRequestsForm.java:43)
	at com.microsoft.alm.plugin.idea.git.ui.pullrequest.VcsPullRequestsController.<init>(VcsPullRequestsController.java:21)
	at com.microsoft.alm.plugin.idea.git.extensions.VcsPullRequestContentProvider.<init>(VcsPullRequestContentProvider.java:33)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.intellij.util.pico.InstantiatingComponentAdapter.newInstance(CachingConstructorInjectionComponentAdapter.java:294)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.doGetComponentInstance(CachingConstructorInjectionComponentAdapter.java:88)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.instantiateGuarded(CachingConstructorInjectionComponentAdapter.java:66)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.getComponentInstance(CachingConstructorInjectionComponentAdapter.java:48)
	at com.intellij.openapi.vcs.changes.ui.ChangesViewContentEP.newClassInstance(ChangesViewContentEP.java:136)
	at com.intellij.openapi.vcs.changes.ui.ChangesViewContentEP.getInstance(ChangesViewContentEP.java:95)
	at com.intellij.openapi.vcs.changes.ui.VcsToolWindowFactory$createExtensionContent$$inlined$apply$lambda$1.invoke(VcsToolWindowFactory.kt:102)
	at com.intellij.openapi.vcs.changes.ui.VcsToolWindowFactory$createExtensionContent$$inlined$apply$lambda$1.invoke(VcsToolWindowFactory.kt:28)
	at com.intellij.openapi.vcs.changes.ui.ChangesViewContentManager$ContentProvidersListener.selectionChanged(ChangesViewContentManager.kt:168)
	at jdk.internal.reflect.GeneratedMethodAccessor144.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.intellij.util.EventDispatcher.dispatchVoidMethod(EventDispatcher.java:127)
	at com.intellij.util.EventDispatcher.access$000(EventDispatcher.java:21)
	at com.intellij.util.EventDispatcher$1.invoke(EventDispatcher.java:85)
	at com.sun.proxy.$Proxy118.selectionChanged(Unknown Source)
	at com.intellij.ui.content.impl.ContentManagerImpl.fireSelectionChanged(ContentManagerImpl.java:579)
	at com.intellij.ui.content.impl.ContentManagerImpl.addSelectedContent(ContentManagerImpl.java:397)
	at com.intellij.ui.content.impl.ContentManagerImpl$1.run(ContentManagerImpl.java:487)
	at com.intellij.ui.content.impl.ContentManagerImpl.lambda$setSelectedContent$4(ContentManagerImpl.java:500)
	at com.intellij.openapi.util.ActionCallback$ExecutedExecutionCallback.doWhenExecuted(ActionCallback.java:177)
	at com.intellij.openapi.util.ActionCallback.doWhenDone(ActionCallback.java:111)
	at com.intellij.openapi.util.ActionCallback.doWhenProcessed(ActionCallback.java:129)
	at com.intellij.ui.content.impl.ContentManagerImpl.setSelectedContent(ContentManagerImpl.java:500)
	at com.intellij.ui.content.impl.ContentManagerImpl.setSelectedContentCB(ContentManagerImpl.java:454)
	at com.intellij.ui.content.impl.ContentManagerImpl.setSelectedContentCB(ContentManagerImpl.java:443)
	at com.intellij.ui.content.impl.ContentManagerImpl.setSelectedContent(ContentManagerImpl.java:437)
	at com.intellij.openapi.wm.impl.content.ContentTabLabel.selectContent(ContentTabLabel.java:243)
	at com.intellij.openapi.wm.impl.content.ContentTabLabel$2.execute(ContentTabLabel.java:138)
	at com.intellij.util.ui.BaseButtonBehavior$MyMouseListener.execute(BaseButtonBehavior.java:173)
	at com.intellij.util.ui.BaseButtonBehavior$MyMouseListener.mouseReleased(BaseButtonBehavior.java:156)
	at java.desktop/java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:297)
	at java.desktop/java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:297)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6651)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3345)
	at java.desktop/java.awt.Component.processEvent(Component.java:6416)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5026)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4858)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2773)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4858)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:778)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:751)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:749)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:748)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:974)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.java:912)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:844)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:449)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:739)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:448)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:832)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:502)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

I am also seeing issue #292 in the logs.

**Comment by ForNeVeR** [comment]
Thanks for your report!

Both of your issues should already be fixed in the recently released v1.159.0.

Because of the way plugin is bundled into Rider, it may be difficult to update the plugin (because auto update doesn't work for v1.158 and earlier). For now, you may either wait for Rider 2020.1 EAP 7 release (which includes a fresh plugin version, and also relaxes the update mechanism so auto updates will work in future), or update manually: download the latest plugin version from [the plugin portal](https://plugins.jetbrains.com/plugin/7981-azure-devops) and install it via **Settings → Plugins → Install Plugin from Disk**.

![image](https://user-images.githubusercontent.com/92793/78248482-e8948800-7516-11ea-8e16-a610ef320717.png)

**Comment by caveman-dick** [comment]
@ForNeVeR Brilliant thanks that fixed it! :)
